restapi/stories.go

```diff
@@ -129,7 +129,7 @@ func (api api) CreateStoriesFormData(w http.ResponseWriter, r *http.Request) ren
 	if err != nil {
 		return rest.ErrInvalidRequest(api.logger, "Error getting the image.", err)
 	}
-	res, err := api.s3upload(file, h.Size, h.Filename)
+	ImageURL, err := api.uploadImageTos3(file, h.Size, h.Filename)
```

Comment:
```suggestion
	imageURL, err := api.uploadImageTos3(file, h.Size, h.Filename)
```

prefer lowercase here